README.md

```diff
@@ -23,4 +25,4 @@ VERSION="1.3.0"
 ## Deploy staging version (when version in pom is -SNAPSHOT)
 
 From a laptop with access to deployment cluster
-`make jar-signing authenticode-signing`
```

Comment:
I'd rather fix this in the make file directly:

```
diff --git a/Makefile b/Makefile
index 0590c81..02d3d40 100644
--- a/Makefile
+++ b/Makefile
@@ -2,12 +2,12 @@
 .PHONY: clean jar-signing-service authenticode-signing macos
 
 jar-signing:
-	./mvnw verify -am -pl webservice/signing/jar
+	./mvnw clean verify -am -pl webservice/signing/jar
 	./webservice/service-deployment.sh webservice/signing/jar/default.jsonnet
 	./webservice/service-deployment.sh webservice/signing/jar/jce.jsonnet
 
 authenticode-signing:
-	./mvnw verify -am -pl webservice/signing/windows
+	./mvnw clean verify -am -pl webservice/signing/windows
 	./webservice/service-deployment.sh webservice/signing/windows/service.jsonnet
 
 macos:

```

Reply:
make clean a dependency for each webservice deployment, added an explicit all target.